src/main/java/com/lucidworks/spark/SolrRelation.java

```diff
@@ -63,8 +74,17 @@ public SolrRelation(SQLContext sqlContext, Map<String,String> config, DataFrame
     solrQuery.set("collection", collection);
     if (dataFrame != null) {
       schema = dataFrame.schema();
-    } else {
-      schema = solrRDD.getQuerySchema(solrQuery);
+    }
+    else {
+      if(!preserveSchema.equals("Y")) {
```

Comment:
what's the convention for other Spark properties for indicating "true"?  Do they pass in "Y/N", or "true/false"?